src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -188,34 +197,38 @@ def update(self, time, randvar, data, **kwargs):
             which is the crosscov between input RV and measured RV.
             The crosscov does not relate to the updated RV!
         """
-        meas_rv, info = self.measure(time, randvar, **kwargs)
+        # if required, preconditioning can be leveraged here, similarly to `smooth_step`
```

Comment:
shall this comment stay? I didn't implement this, because as of now, we only need preconditioning for smoothing.

Reply:
I think it's fine to not have it implemented, as you say that it should not matter for the forward pass. I'm fine with both keeping and deleting the comment.

Reply:
I will throw a coin

Reply:
removed it.